samples/CallWithChat/src/app/views/HomeScreen.tsx

```diff
@@ -135,9 +139,58 @@ export const HomeScreen = (props: HomeScreenProps): JSX.Element => {
                 label={'Meeting Link'}
                 required
                 placeholder={'Enter a Teams meeting link'}
-                onChange={(_, newValue) => newValue && setTeamsLink({ meetingLink: newValue })}
+                onChange={(_, newValue) => {
+                  newValue ? setMeetingLocator({ meetingLink: newValue }) : setMeetingLocator(undefined);
+                }}
               />
             )}
+            {
+              /* @conditional-compile-remove(meeting-id) */ teamsCallChosen && (
+                <Text className={teamsItemStyle} block variant="medium">
+                  <b>Or</b>
+                </Text>
+              )
+            }
+            {
+              /* @conditional-compile-remove(meeting-id) */ teamsCallChosen && (
+                <TextField
+                  className={teamsItemStyle}
+                  iconProps={{ iconName: 'MeetingId' }}
+                  label={'Meeting Id'}
+                  required
+                  placeholder={'Enter a meeting id'}
+                  onChange={(_, newValue) => {
+                    const passcode =
+                      meetingLocator && 'passcode' in meetingLocator ? meetingLocator.passcode : undefined;
+                    newValue
+                      ? setMeetingLocator({ meetingId: newValue, passcode: passcode })
+                      : setMeetingLocator(undefined);
+                  }}
+                />
+              )
+            }
+            {
+              /* @conditional-compile-remove(meeting-id) */ teamsCallChosen && (
+                <TextField
+                  className={teamsItemStyle}
+                  iconProps={{ iconName: 'passcode' }}
+                  label={'Passcode'}
+                  placeholder={'Enter a meeting passcode'}
+                  onChange={(_, newValue) => {
+                    const meetingId = meetingLocator && 'meetingId' in meetingLocator ? meetingLocator.meetingId : '';
+                    // meeting id is required, but passcode is not
+                    setMeetingLocator({ meetingId: meetingId, passcode: newValue });
```

Comment:
This doesn't capture if meetingId is null at this point. Meaning it could set the locator to:
```
{
  meetingId: '',
  passcode: '<some value>'
```

Thus enabling the start call button, but where the join will fail.

Let's make sure we don't allow users to get into bad paths